sa/sa_test.go

```diff
@@ -383,6 +383,10 @@ func TestAddCertificate(t *testing.T) {
 	test.Assert(t, certificateStatus.Status == core.OCSPStatusGood, "OCSP Status should be good")
 	test.Assert(t, certificateStatus.OCSPLastUpdated.IsZero(), "OCSPLastUpdated should be nil")
 
+	if CertStatusOptimizationsMigrated {
```

Comment:
Once the migration is applied and the flag is set to true then the unit test will ensure that the field is updated appropriately.